packages/util/src/index.ts

```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-export * from '@polkadot/util';
-export * from '@polkadot/util-crypto';
+// export * from '@polkadot/util';
```

Comment:
any action needed here?

Reply:
I will re-add this export... my build used to fail when I had this...

Reply:
added it back and updated typescript version

Reply:
have removed 
````
export * from '@polkadot/util-crypto';
````

The issue here is both of these packages internally has already exported a member name 'packageInfo'... which creates ambiguity... I couldn't find a way where I can export everything except one member... either export the list of member from every package over * or may be need a script to filter to filter packageInfo... will look at this

Reply:
https://github.com/Microsoft/TypeScript/pull/5148/files
This is introduced by typescript to avoid ambiguity when member of same name exist in the exported package...
exported individual package from @polkadot/util-crypto..